### PR TITLE
Add CLI doc page for dumping and uploading cluster wide secrets.

### PR DESCRIPTION
**Pull request by adityadani**

### CODE REVIEW

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Dump and Upload cluster-wide secrets
+linkTitle: Dump and Upload cluster-wide secrets
+keywords: portworx, pxctl, command-line tool, cli, reference, alerts, monitoring, encrypted-volumes, encryption, secrets
+description: Migrate cluster-wide secrets used for encrypting volumes between clusters. 
+weight: 18
+---
+
+_Portworx_ provides the capability to encrypt volumes using cluster-wide secrets. Cluster-wide secret is a unique secret
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
_Portworx_ provides the capability to encrypt volumes using cluster-wide secrets. A cluster-wide secret is a unique secret
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+for a cluster that can be used as a default key for encrypting your volumes. However this poses a problem while migrating
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
for a cluster that can be used as a default key for encrypting your volumes. However, this poses a problem while migrating 
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+migrated encrypted volume.
+
+The following set of commands will help you dump the cluster-wide secret from one cluster and upload the same secret to
+a different cluster. Once the cluster-wide secret is uploaded to the destination cluster, encrypted volumes using the cluster-wide
+secret can be migrated to the destination cluster.
+
+
+### Dumping cluster-wide secret
+
+To dump the cluster-wide secret run the following command
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
Run the following command to dump the cluster-wide secret:
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+```text
+pxctl secrets  dump-cluster-wide-secret
+```
+
+```output
+Following are the details about the cluster-wide secret for this cluster:
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: Is this part of the output? or is the output tag misplaced?

```suggestion
The following example provides details the cluster-wide secret for this cluster:
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Secret ID (--secret_id): shailvitest
+Secret value (--secret_value): AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
+
+On the destination cluster run the following command:
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
Run the following command on the destination cluster:
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+ /opt/pwx/bin/pxctl secrets upload-cluster-wide-secret --secret_id shailvitest --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
+
+```
+
+The _dump_ command also spits out the corresponding upload command that needs to be executed on the destination cluster.
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: Is there an example of the `dump` command's syntax? I'm not seeing it. 
```suggestion
The `dump` command also spits out the corresponding upload command that needs to be executed on the destination cluster.
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+### Upload cluster-wide secret
+
+Use the command from the output of the `dump-cluster-wide-secret` command on the destination cluster to upload the cluster-wide secret
```
Comment (nathan-px) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: Is this correct? I'm trying to add clarity to the command that results from the dump command.

```suggestion
The `dump-cluster-wide-secret` command outputs an `upload-cluster-wide-secret` command. Use this command on the destination cluster to upload the cluster-wide secret:
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+```output
+Following are the details about the cluster-wide secret for this cluster:
+
+Secret ID (--secret_id): shailvitest
+Secret value (--secret_value): AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
+
+On the destination cluster run the following command:
+
+ /opt/pwx/bin/pxctl secrets upload-cluster-wide-secret --secret_id shailvitest --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```
Comment (vilasd) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: Please rename the id to something like demo

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+To dump the cluster-wide secret run the following command
+
+```text
+pxctl secrets  dump-cluster-wide-secret
+```
+
+```output
+Following are the details about the cluster-wide secret for this cluster:
+
+Secret ID (--secret_id): shailvitest
```
Comment (vilasd) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: demotest

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+The _dump_ command also spits out the corresponding upload command that needs to be executed on the destination cluster.
+
+### Upload cluster-wide secret
+
+Use the command from the output of the `dump-cluster-wide-secret` command on the destination cluster to upload the cluster-wide secret
+
+```text
+pxctl secrets upload-cluster-wide-secret --secret_id shailvitest --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```
Comment (vilasd) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]: change the names here too

File: content/reference/CLI/dump-upload-cluster-wide-secret.md (hunk 4 above)
Comment (adityadani) [reply content/reference/CLI/dump-upload-cluster-wide-secret.md]: It is a part of the output.

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+Run the following command to dump the cluster-wide secret:
+
+```text
+pxctl secrets  dump-cluster-wide-secret
+```
+
+```output
+Following are the details about the cluster-wide secret for this cluster:
+
+Secret ID (--secret_id): shailvitest
```
Comment (adityadani) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
Secret ID (--secret_id): demo_secret_id
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+Secret value (--secret_value): AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
+
+Run the following command on the destination cluster:
+
+ /opt/pwx/bin/pxctl secrets upload-cluster-wide-secret --secret_id shailvitest --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```
Comment (adityadani) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
 /opt/pwx/bin/pxctl secrets upload-cluster-wide-secret --secret_id demo_secret_id --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```

File: content/reference/CLI/dump-upload-cluster-wide-secret.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+The `dump` command also spits out the corresponding upload command that needs to be executed on the destination cluster.
+
+### Upload cluster-wide secret
+
+The `dump-cluster-wide-secret` command outputs an `upload-cluster-wide-secret` command. Use this command on the destination cluster to upload the cluster-wide secret:
+
+```text
+pxctl secrets upload-cluster-wide-secret --secret_id shailvitest --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```
Comment (adityadani) [inline content/reference/CLI/dump-upload-cluster-wide-secret.md]:
```suggestion
pxctl secrets upload-cluster-wide-secret --secret_id demo_secret_id --secret_value AQIDAHiFBNOMzjedxcurzMsjimuzjkzzx99/lUtayeBPH38SNAF5duc4foWsMAGBmsjOvLPtAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZ9WsX37ZmWVPIjsEAgEQgDs+UIl5k3AK4JxpX4uiw1qQbGwBuGALKIL7rYN8k5daq0q3xnz+EXs8PFltmkNbZGcGfkKelIS2Z5dYhA==
```

**Pull request merged**